cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -1018,7 +1038,67 @@ def setUp(self):
         )
         self.html_unit3.parent = self.vertical2
 
-        groups_list = {
+        c0_url = self.course.id.make_usage_key("vertical", "condition_0_vertical")
+        c1_url = self.course.id.make_usage_key("vertical", "condition_1_vertical")
+        c2_url = self.course.id.make_usage_key("vertical", "condition_2_vertical")
+
+        self.split_test_unit = ItemFactory.create(
```

Comment:
This setup code is very verbose and hence hard to read. Could there be helper methods to simplify some of this logic?